docs/src/docs/next-steps.md

```diff
@@ -51,29 +77,49 @@ code:
   path: ./notebooks
 ```
 
-A minimal ModelKit for distributing a pair of datasets might look like this:
+A minimal ModelKit for distributing a pair of datasets looks like this:
+
 ```yaml
 manifestVersion: v1.0.0
 
+package:
+  authors: Jozu
+
 datasets:
 - name: training data
   path: ./data/train.csv
+  license: Apache-2.0
 - description: validation data
   name: validation data
   path: ./data/validate.csv
 ```
 
-Once you've authored a Kitfile for your AI/ML project you can pack it up and store it in your local or remote repository.
+More information on Kitfiles can be found in the [Overview](./kitfile/kf-overview.md) and [Format](./kitfile/format.md) documentation.
 
-If you don't provide a registry Kit assumes you're operating against your local registry. For example:
+### Using the Kitfile to Pack a ModelKit
+
+When you're done writing the Kitfile, name it `kitfile` without an extension. Now you can use the `kit pack` command to build your ModelKit.
```

Comment:
I think our current "default" name is `Kitfile` with a capital K, though we can change this. If you unpack, you'll get a `Kitfile` instead of a `kitfile`.

Reply:
FWIW `kitfile` worked on my machine as well but I've updated it to say `Kitfile` in the latest commit to be consistent with the pulled result.